### DISCUSSION TITLE
[Design] What is the sum aggregator for a non-existing value?

### DISCUSSION THREAD

**Discussion by anjakefala** [saulpw/visidata#1999] [Ideas]
Kivell has never sold a Pencil.

![Screenshot from 2023-08-16 22-22-45](https://github.com/saulpw/visidata/assets/7489659/90b2267b-9a46-4a6f-8e43-6124ba3b0818)

Previously, his `mean_Pencil` would be `Null`, and the `sum_Pencil` would be 0.

With https://github.com/saulpw/visidata/commit/1c466d830e55988e3830fa2cc60de4a265455f19 now the `sum_Pencil` is similarly `Null`.

![Screenshot from 2023-08-16 22-23-13](https://github.com/saulpw/visidata/assets/7489659/ae26fcf1-041f-40eb-841b-f70b53c9036c)

Which is your preferred behaviour? =)

**Comment by maxim-uvarov** [comment]
`Sum_Pencil = 0` seems to me more logical

**Comment by LaPingvino** [comment]
I think converting null to 0 is trivial, but the second way to do it adds possible information that you can work with that wouldn't be there otherwise... I think both ways have their merit honestly!
